terraform/modules/ignition/ignition.tf

```diff
@@ -66,86 +68,88 @@ data "ignition_systemd_unit" "update-engine" {
 }
 
 data "template_file" "eco-service" {
-  template = "${file("${path.module}/resources/eco.service")}"
+  template = file("${path.module}/resources/eco.service")
 
-  vars {
-    image = "${var.eco_image}"
+  vars = {
+    image = var.eco_image
   }
 }
 
 data "ignition_systemd_unit" "eco" {
   name    = "eco.service"
-  content = "${data.template_file.eco-service.rendered}"
+  content = data.template_file.eco-service.rendered
 }
 
 data "ignition_systemd_unit" "eco-health" {
   name    = "eco-health.service"
-  content = "${file("${path.module}/resources/eco-health.service")}"
+  content = file("${path.module}/resources/eco-health.service")
 }
 
 data "ignition_systemd_unit" "node-exporter" {
   name    = "node-exporter.service"
-  content = "${file("${path.module}/resources/node-exporter.service")}"
+  content = file("${path.module}/resources/node-exporter.service")
 }
 
 data "ignition_file" "eco-config" {
   filesystem = "root"
   path       = "/etc/eco/config.yaml"
-  mode       = 0644
+  mode       = 420
```

Comment:
nit: did they deprecate this syntax? it feels easier to read as 0644 to me as it is the usual chmod syntax.

Reply:
yes ... seems so -- don't see in release notes but there is discussion in issues.